### ISSUE TITLE
Leverage seaborn in figure creation

### ISSUE THREAD

**Issue by tsalo** [Brainhack-Donostia/ica-aroma-org#47]
### Summary
The classification plot code uses seaborn, but doesn't take advantage of the available features. We should review the plotting code and identify places where it can be improved.

### Additional Detail
An example- instead of using `seaborn`'s `hue` argument, the classification plot code duplicates the same lines for the accepted and rejected components. This also requires the code to add fake rows in cases where all components are either rejected or accepted.

### Next Steps
1. Review the classification plot code for unnecessary duplication.
2. Fix up seaborn usage.

**Comment by tsalo** [comment]
Here is an example of duplication that can be eliminated by using the `hue` argument:

https://github.com/Brainhack-Donostia/ica-aroma-org/blob/10f5183671d2a5014b96bc10447ac6d41d9f792b/aroma/plotting.py#L144-L156

**Comment by eurunuela** [comment]
This is a great little task for anyone from Brainhack Donostia joining the project!

**Comment by oesteban** [comment]
> This is a great little task for anyone from Brainhack Donostia joining the project!

Yes!, but I would say this should be contributed directly upstream to ICA-AROMA. I think this re-implementation should focus only on the core of the method.

I also know for experience that the plotting module of ICA-AROMA is not very stable, so you don't want to bring that here. Instead, you want all the ICA-AROMA current users to benefit from improvements making the plotting better.

WDYT?
